syntax = "proto2";
package Trd_UnlockTrade;

import "Common.proto";

message C2S
{
	required bool unlock = 1; //true解锁，false锁定
	optional string pwdMD5 = 2; //交易密码的MD5转16进制小写，解锁必须要填密码，锁定忽略
}

message S2C
{
	
}

message Request
{
	required C2S c2s = 1;
}

message Response
{
	//以下3个字段每条协议都有，注释说明在InitConnect.proto中
	required int32 retType = 1 [default = -400];
	optional string retMsg = 2;
	optional int32 errCode = 3;
	
	optional S2C s2c = 4;
}
